src/backend/base/langflow/components/data/api_request.py

```diff
@@ -257,12 +269,12 @@ def add_query_params(self, url: str, params: dict) -> str:
         query = dict(parse_qsl(url_parts[4]))
         query.update(params)
         url_parts[4] = urlencode(query)
+        print(url_parts)
```

Comment:
I think this should be removed

Reply:
You're absolutely right. This is likely leftover debug output from the development phase that I forgot to remove in the final version. I'll clean it up—thanks for catching that!